src/azul/indexer/log_forwarding_controller.py

```diff
@@ -32,8 +32,6 @@ def s3(self) -> LogForwardingService:
         return S3AccessLogForwardingService()
 
     def _forward_logs(self, event: chalice.app.S3Event, service: LogForwardingService) -> None:
-        # FIXME: Create alarm for log forwarding failures
-        #        https://github.com/DataBiosphere/azul/issues/4997
```

Comment:
It's unusual for a commit to do nothing besides removing a FIXME comment, so I suggest updating the title for the first commit to "Remove FIXME for duplicate issue (#4997)" to make it clear why we're doing this.

Reply:
You could also squash this into the main commit, and include both issue ref's in the same commit title.